LocalSettings.php

```diff
@@ -4416,7 +4416,7 @@
 
 // ManageWiki settings
 require_once __DIR__ . '/ManageWikiExtensions.php';
-$wi->disabledExtensions = [];
+$wi->disabledExtensions = [ 'pageforms' ];
```

Comment:
This won't disable. Needs to be PageForms, the same value as the 'name' field in ManageWiki I believe.